test/tsconfig-test.json

```diff
@@ -1,9 +1,9 @@
 {
     "extends": "../tsconfig.json",
     "compilerOptions": {
-        "composite": true,
         "outDir": "../",
-        "types": ["mocha", "chai", "sinon", "webpack-env", "node"]
+        "types": ["mocha", "chai", "sinon", "webpack-env", "node"],
+        "declaration": false
```

Comment:
Not emitting declarations at testing times means that already build declarations from build time are not overwritten, which `tsc` throws on.